test/harness.js

```diff
@@ -0,0 +1,43 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+'use strict'
+
+const dbServer = require('../fxa-auth-db-server')
+const backendTests = require('../fxa-auth-db-server/test/backend')
+const config = require('../config')
+const log = require('./lib/log')
+const P = require('bluebird')
+
+const DBS = {
+  mem: require('../lib/db/mem'),
+  mysql: require('../lib/db/mysql')
+}
+
+exports.dbTests = function dbTests(backend) {
+  const DB = DBS[backend](log, dbServer.errors)
+
+  backendTests.dbTests(config, DB)
+}
+
+exports.remote = function remote(backend) {
+  const DB = DBS[backend](log, dbServer.errors)
+
+  backendTests.remote(config, () => {
+    return DB.connect(config)
+    .then(function (newDb) {
+      const db = newDb
+      const server = dbServer.createServer(db)
```

Comment:
I still need to figure out how to attach `node --inspect` debuggers to these magical `createServer` calls in db and auth server